src/index.html

```diff
@@ -11,7 +11,151 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+    
+      <fieldset class="group">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input 
+           type="text"
+           name="surname"
+           autocomplete="off"
+          >
```

Comment:
```suggestion
          <input 
            type="text"
            name="surname"
            autocomplete="off"
          >
```
should be 2 spaces, check in other places